examples/data_collect/main.cpp

```diff
@@ -275,7 +280,10 @@ int main(int argc, char *argv[]) {
     System system;
     std::vector<std::shared_ptr<Camera>> cameras;
 
-    if (!ip.empty()) {
+    if (!ip.empty() && useNetLinkTest) {
+        ip = "ip:" + ip + ":netlinktest";
+        system.getCameraList(cameras, ip);
+    } else if (!ip.empty()) {
         ip = "ip:" + ip;
```

Comment:
also this, you can call this whenever ip variable is not empty. And additionally, when useNetLinkTest is true, just append to the ip variable a new string